CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Braintree Android SDK Release Notes
 
+## unreleased
+* PayPalNativeCheckout
+  * Changing the way the  `UrlResponseData` inside the PayPalNativeCheckoutClient is set
```

Comment:
I don't know that we need a changelog for this unless there is a merchant facing impact to this change. Is there a behavior they would see before/after this change?